experimental/GITFans/test/runtests.jl

```diff
@@ -52,7 +52,10 @@
            BitSet([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]),
            BitSet([1])]
 
-    intergraph = Polymake.graph.graph_from_edges(collect(edges));
+    orbit_graph = Graph{Undirected}(length(hash_list));
+    [add_edge!(orbit_graph, e...) for e in edges];
```

Comment:
Lots of trailing semicolons here and elsewhere - remove?

Reply:
I am using parts of these tests for copy & paste in a Julia session, and the output of intermediate steps is uninteresting. In particular the commands mentioned here produce a full screen of useless output.
What is the problem with suppressing output?
I would argue that output should not be shown except if there is a reason to show it.

Reply:
We do not add `;` to every line of all files, just in case someone wants to copy them to the REPL.

Reply:
Hm, I don't agree on this point. I can see that doctests have some REPL flavour, but not test files. Some of them require  some setup (which might not even be part of that file), and often the calls are "non-local" (e.g. require variables defined a hundered lines before).

Anyway, this should not hold up this PR. We are inconsistent anyway.

Reply:
What is the difference in processing a test file with or without a semicolon at the end of a command? Is perhaps the `show` method of the return value called in the case that there is no semicolon, and thus one tests also whether this method does not run into an error? That would be a reason to force that there is no semicolon.